app/messaging/send-audit.js

```diff
@@ -326,6 +326,33 @@ const sendViewToAudit = async (pk, type, subject, details, { username, displayna
   await sendEvent(event)
 }
 
+const sendLoginToAudit = async ({ username, displayname }, userAgent) => {
```

Comment:
There is a method sendViewToAudit() line 305 in this file already. Can we re-use that call with appropriate params